tests/meshcop/commissioner.cpp

```diff
@@ -201,32 +103,54 @@ void HandleRelayReceive(const Coap::Resource &aResource, const Coap::Message &aM
          LengthOf(payload, requestTlv) < length;
          requestTlv = requestTlv->GetNext())
     {
-        switch (requestTlv->GetType())
+
+        tlvtype = requestTlv->GetType();
+        switch (tlvtype)
         {
         case Meshcop::kJoinerDtlsEncapsulation:
             struct sockaddr_in addr;
             memset(&addr, 0, sizeof(addr));
             addr.sin_family = AF_INET;
             addr.sin_port = htons(kPortJoinerSession);
 
-            ret = sendto(context.mSocket, requestTlv->GetValue(), requestTlv->GetLength(),
-                         0, reinterpret_cast<const struct sockaddr *>(&addr), sizeof(addr));
+            otbrLog(OTBR_LOG_INFO, "Encapsulation: %d bytes for port: %d",
+                    requestTlv->GetLength(), kPortJoinerSession);
+            {
+                char buf[50];
+                inet_ntop(AF_INET, &addr.sin_addr, buf, sizeof(buf));
+                otbrLog(OTBR_LOG_INFO, "DEST: %s", buf);
+            }
+            ret = sendto(context.mSocket,
+                         requestTlv->GetValue(),
+                         requestTlv->GetLength(),
+                         0,
+                         reinterpret_cast<const struct sockaddr *>(&addr),
+                         sizeof(addr));
+            if (ret < 0)
+            {
+                otbrLog(OTBR_LOG_ERR, "relay receive, sendto() fails with %d", ret);
```

Comment:
sendto() returns the number of byte sent, or -1 on error, the negative test is sufficent because you cannot send a negative number of bytes.

print errno - good catch.